sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HivePartitionFilteringSuite.scala

```diff
@@ -577,6 +582,60 @@ class HivePartitionFilteringSuite(version: String)
       dateStrValue)
   }
 
+  test("getPartitionsByFilter: substr(chunk,0,1)=a") {
```

Comment:
nit: could we prepend JIRA number to these new tests?

Reply:
I probably remember that UT does not need to add a jira number if it is not for bug fixes.
I still add the jira number here.

Reply:
hm maybe you are right - sorry for the confusion.